.tekton/clowder-pull-request.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "master"
```

Comment:
this needs to be `master` in order to be triggered, as it defines the target branch against the PR is going to be merged